### DISCUSSION TITLE
handbook/web-testing/serenity-js-web-modules/

### DISCUSSION THREAD

**Discussion by giscus** [serenity-js/serenity-js#1856] [Comments]
# handbook/web-testing/serenity-js-web-modules/

While the Screenplay Pattern is a behavioural pattern,

https://serenity-js.org/handbook/web-testing/serenity-js-web-modules/

**Comment by mSaurabhPS** [comment]
I am having a really hard time understanding the Page Element Query Language and not able to find much support online. Id really appreciate if you can create a section just for that with examples around how to select a button, form elements and examples of sorts. Thanks a lot

**Reply by jan-molak** [reply]
Hey @mSaurabhPS! I'm guessing you've already seen the [Page Element Query Language](https://serenity-js.org/handbook/web-testing/page-element-query-language/) article and the examples there?
 
I wonder if it could be useful to have a dedicated tutorial on "Working with forms". Is that what you had in mind?
